### PR TITLE
Add documentation around DNS failover and static site stub

### PR DESCRIPTION
**Pull request by hectcastro**
More documentation for the Route 53 DNS failover requirements, and a proposed home for the contents of the static S3 website.

### CODE REVIEW

**Comment by hectcastro** [comment]
@azavea-bot rebuild

File: deployment/README.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ $ cd deployment
 $ pip install -r deployment/requirements.txt
 ```
 
+## S3 Website Failover
+
+Route 53 DNS failover is configured by the following deployments steps in the event that all application servers fail simultaneously. The contents of the failover site are located in `src/failover_website` from the project root.
```
Comment (jwalgran) [inline deployment/README.md]: deployments -> deployment

File: deployment/README.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ $ cd deployment
...
+
```
Comment (jwalgran) [inline deployment/README.md]: The text mentions "configured by the following deployment steps" but there are no steps listed after the paragraph.

**Comment by jwalgran** [comment]
:+1:

File: deployment/README.md (hunk 2 above)
Comment (hectcastro) [inline deployment/README.md]: I was referring to the steps after this section. Need to be clearer.

**Pull request merged**